docker/docker-entrypoint.sh

```diff
@@ -15,23 +15,36 @@ map_uidgid() {
 	fi
 }
 
+map_folders() {
+	# Export these so they can be used in docker-prepare.sh
+	export DATA_DIR="${PAPERLESS_DATA_DIR:-/usr/src/paperless/data}"
+	export MEDIA_ROOT_DIR="${PAPERLESS_MEDIA_ROOT:-/usr/src/paperless/media}"
+}
+
 initialize() {
+	# Change the user and group IDs if needed
 	map_uidgid
 
-	for dir in export data data/index media media/documents media/documents/originals media/documents/thumbnails; do
```

Comment:
It's probably good to make sure `/usr/src/paperless/export` exists, in case other things have it hardcoded.  But since the exporter already requires the folder as a parameter, it's probably alright.

Reply:
Ah, since it's a volume in the Dockerfile, it will always exist.  But might as well keep it consistent with before for now.

Reply:
Ah, since it's a volume in the Dockerfile, it will always exist.  But might as well keep it consistent with before for now.